app/views/events/show.html.erb

```diff
@@ -53,15 +53,14 @@
   </div>
 
   <% if !@event.guests.empty? %>
-    <% count = @event.guests.where(status: "Accept").or(@event.guests.where(status: "Pending")).count %>
+    <% count = @event.guests.where(status: "Accept").count %>
```

Comment:
I may be wrong, but since you already did that, can't we just use count = @confirmed_guests.count?